Tella/Scenes/HomeView/ViewModel/HomeViewModel.swift

```diff
@@ -28,16 +28,24 @@ class HomeViewModel: ObservableObject {
     }
     
     func getServersList() {
-        
-        self.appModel.vaultManager.tellaData.servers.sink { result in
             
-        } receiveValue: { serverArray in
-            self.serverDataItemArray.removeAll()
-            if !serverArray.isEmpty {
-                self.serverDataItemArray.append(ServerDataItem(servers: serverArray, serverType: .tellaUpload))
-            }
-        }.store(in: &subscribers)
-    }
+            self.appModel.vaultManager.tellaData.servers.sink { result in
+                
+            } receiveValue: { serverArray in
+                self.serverDataItemArray.removeAll()
+                if !serverArray.isEmpty {
+                    // here i group all the tella servers in one array and the third party services in diferents arrays
+                    let thirdPartyConnections = serverArray.filter { mapServerTypeFromInt($0.serverType) != .tella }
```

Comment:
@rob-horizontal-iOS Do you mean with `thirdPartyConnections` Uwazi ?

Reply:
I write that part @dhekra-rouatbi, with third party connections I mean all the other servers integrations that are not tella. Because in the future we will expect to have multiple types of servers connections. 

I thought that we will show a different card for each connection so except for tella that have all the connections in one card but after talking with caro I realized that we need to group all connections in the same server type. I could create just one array of serverConnections and distinguish it with the server type